pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1342,6 +1345,9 @@ func (n *NetworkPolicyController) syncAppliedToGroup(key string) error {
 				memberSetByNode[entityNodeKey] = entitySet
 				appGroupNodeNames.Insert(entityNodeKey)
 			}
+			for _, node := range nodes {
+				appGroupNodeNames.Insert(node.Name)
```

Comment:
Like Pod case, we should fill `memberSetByNode` to only send the Node itself to a Node. Otherwise it will have to resend the whole members everytime a Node joins or leaves the group.
Please have an unit test to validate it.